adventure/migrations/0001_initial.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.1.1 on 2018-10-01 7:03
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Player',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
Python actually has a hard rule about line length. I've posted a link to it below. This is something you should keep in mind while you configure your environments. Every language has it's own rules. Python has the pythonic way and the wrong way. If you enjoy the language I would recommend studying up!

https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
HINT: switch your space values from 4 down to 2 for the python language. It will give you more room for your indents. But you will definetly need an auto formatter then.